src/content_script.js

```diff
@@ -70,9 +70,29 @@ function downloadThread({ as = Format.PNG } = {}) {
   const pixelRatio = window.devicePixelRatio;
   const minRatio = as === Format.PDF ? 2 : 2.5;
   window.devicePixelRatio = Math.max(pixelRatio, minRatio);
-  html2canvas(elements.thread, {
-    letterRendering: true,
-  }).then(async function (canvas) {
+  html2canvas(elements.thread, 
+    {
+      letterRendering: true,
+      onclone: function (cloneDoc) {
+        //Make small fix of position to all the text containers
+        let listOfTexts = cloneDoc.getElementsByClassName('min-h-[20px]');
+
+        for (let i = 0, leng = listOfTexts.length; i < leng; i++) {
+          let text = listOfTexts[i];
+          text.style['position'] = 'relative';
+          text.style['top'] = '-8px';
+        }
+
+        //Delete copy button from code blocks
+        let listOfCopyBtns = cloneDoc.querySelectorAll('button.flex');
+
+        for (let i = 0, leng = listOfCopyBtns.length; i < leng; i++) {
+          let button = listOfCopyBtns[i];
+          button.remove();
+        }
+      }
+    }
+  ).then(async function (canvas) {
```

Comment:
Looks good! What do you think about using `forEach`?

```suggestion
  html2canvas(elements.thread, {
    letterRendering: true,
    onclone: function (cloneDoc) {
      // Make small fix of position to all the text containers
      let listOfTexts = cloneDoc.getElementsByClassName("min-h-[20px]");
      Array.from(listOfTexts).forEach((text) => {
        text.style["position"] = "relative";
        text.style["top"] = "-8px";
      });
      // Delete copy button from code blocks
      let listOfBtns = cloneDoc.querySelectorAll("button.flex");
      Array.from(listOfBtns).forEach((btn) => btn.remove());
    },
  }).then(async function (canvas) {
```

Reply:
Done!